programs/ambassadors/ambassadors-schema.json

```diff
@@ -0,0 +1,77 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "array",
```

Comment:
Can we add the description keyword to provide a basic description of each field?